src/registrar/templates/domain_request_review.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'domain_request_form.html' %}
+{% extends '(request):.html' %}
```

Comment:
Q: This looks fancy, I assume it interpolates the url for the path name in url defs?

Reply:
.....I actually do not remember making this change.  It breaks testing anyways, so I'm reverting it.